### PR TITLE
add hint recipe

### PR DESCRIPTION
**Pull request by suwangbio**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
Nicely done @suwangbio ! Are you a member of bioconda yet? If not I'll add you so you can merge this.

**Comment by suwangbio** [comment]
No, I am not. Please add me as a member.

Thanks,
Su

**Comment by dpryan79** [comment]
Welcome @suwangbio !

File: recipes/hint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/d4/ba/0391dddacaa66b599cff112bc2bed7e0f915ef4aeaf6bf7619368eff3209/HiNT-Package-2.0.2.tar.gz
+  sha256: a57d9ed2e1954cd174aa9a0748f7af56d80c9cd9b4b4deb44a89cefbe4301ecd
+
+build:
+  number: 0
+  noarch: python
+  #script: $PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir .
```
Comment (bgruening) [inline recipes/hint/meta.yaml]: @suwangbio is that not working? I would prefer this line and remove the buid.sh script altogether.

File: recipes/hint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{% set version = "2.0.1" %}
```
Comment (bgruening) [inline recipes/hint/meta.yaml]: This version is not the same as you download? It should be 2.0.2

File: recipes/hint/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - pairtools >=0.2.2
+    - pandas >=0.23.0
+    - scikit-learn >=0.19.1
+    - argparse >=1.1
+    - openjdk >=8
+    - samtools 1.3.1
+    - bwa >=0.7.16
+    - r-mgcv
+    - r-strucchange
+    - r-Cairo
```
Comment (bgruening) [inline recipes/hint/meta.yaml]: r-cairo, I'm wondering why this is working at all :(

File: recipes/hint/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools
+
+  run:
+    - python >=3.5
+    - perl >=5
+    - R >=3.4
```
Comment (bgruening) [inline recipes/hint/meta.yaml]: `r-base`

**Pull request merged**